docs/ai/quickstarts/quickstart-openai-generate-images.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Quickstart - Generate images using Azure AI with .NET
+description: Create a simple app using the .NET Azure OpenAI SDK to generate postal card images.
+ms.date: 03/04/2024
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: fboucher
+ms.author: frbouche
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code.
+---
+
+# Build an Azure AI chat app with .NET
+
+Get started with the .NET Azure OpenAI SDK by creating a simple .NET 8 console chat application. The application will run locally and use the OpenAI `dell-e-3` model to generate postal card and invite your friends for a hike! Follow these steps to provision Azure OpenAI and learn how to use the .NET Azure OpenAI SDK.
+
+[!INCLUDE [download-alert](includes/prerequisites-and-azure-deploy.md)]
+
+## Trying Generate Hiking Images sample
+
+1. From a terminal or command prompt, navigate to the `05-HikeImages` directory.
+2. It's now time to try the console application. Type in the following to run the app:
+
+    ```dotnetcli
+    dotnet run
+    ```
+
+    If you get an error message the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
+
+## Understanding the code
+
+Our application uses the `Azure.AI.OpenAI` client SDK, which is available on [NuGet](https://www.nuget.org/packages/Azure.AI.OpenAI), to send and receive requests to an Azure OpenAI service deployed in Azure.
+
+The `OpenAIClient` class facilitates the requests and responses. `ChatCompletionOptions` specifies parameters of how the model will respond.
+
+```csharp
+var openAIClient = new OpenAIClient(endpoint, credentials);
+
+var completionOptions = new ChatCompletionsOptions
+{
+    MaxTokens = 400,
+    Temperature = 1f,
+    FrequencyPenalty = 0.0f,
+    PresencePenalty = 0.0f,
+    NucleusSamplingFactor = 0.95f, // Top P
+    DeploymentName = openAIDeploymentName
+};
+```
+
+The entire application is contained within the **Program.cs** file. The first several lines of code loads up secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.
+
+```csharp
+// == Retrieve the local secrets saved during the Azure deployment ==========
+var config = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
```

Comment:
I feel like I've suggested this already, which means we should either have this in an include, and share it, or differentiate this content somehow.

```suggestion
var config = new ConfigurationBuilder()
    .AddUserSecrets<Program>()
    .Build();

```